zict/lmdb.py

```diff
@@ -120,7 +135,7 @@ def __contains__(self, item: object) -> bool:
         key, value = item  # type: ignore
         try:
             v = self._mapping[key]
-        except (KeyError, AttributeError):
+        except KeyError:
```

Comment:
Why not `AttributeError` anymore?

Reply:
See `LMDB.__getitem__`. Before, non-string keys would raise AttributeError (from `_encode_key`); now they raise KeyError.